db/seeds.rb

```diff
@@ -63,6 +64,71 @@
 description: "Deep inside an age-old mountain, clamber into a rustic mine train and make your way up a steep incline." 
 )
 
+### Review ###
+review_1 = Review.find_or_create_by(
+  title: 'Fun kids’ ride',
+  body: 'Emerge from the ocean’s depths in time to celebrate Ariel’s happily ever after in a spectacular fairytale finale.',
+  rating: 5,
+  user_id: 1,
+  ride_id: 1 
```

Comment:
rather than using integers for association, we should really be associating objects. This is making an assumption that there will be a user with an id of 1, etc. In place of `ride_id: 1` for example, we should instead say `ride: ride_1` calling on the variable you have defined above! (users can be created in this seeds file by providing them with an email and password properties)